language/move-stdlib/templates/overview.md

```diff
@@ -0,0 +1,64 @@
+# Diem Framework Modules
```

Comment:
Currently this is just a straight copy of `diem-framework/modules/overview_template.md`. I'll write one for Move if we do decide to let `move-stdlib` to manage its own documentation.